package.json

```diff
@@ -5,11 +5,8 @@
   },
   "description": "Strongly typed Cap'n Proto implementation for the browser and Node.js using TypeScript",
   "devDependencies": {
-    "@types/benchmark": "1.0.30",
```

Comment:
these don't need to be added here, `lerna bootstrap --hoist` effectively automatically takes care of this for you.